### DISCUSSION TITLE
Crazyflies cannot work together with other VICON objects

### DISCUSSION THREAD

**Discussion by vietanhle0101** [USC-ACTLab/crazyswarm#697] [Q&A]
I have trouble when using crazyflie with VICON while some other VICON objects are used together.
When I remove the other VICON objects than drones (uncheck in VICON, only check ones for drones), it works well.

Do you have any idea to help me fix this issue? Thanks.
Note: I am using ROS noetic.

![Screenshot from 2022-09-26 10-56-56](https://user-images.githubusercontent.com/51708449/192311776-4af41593-9b77-4615-b048-175201924cfa.png)

**Comment by whoenig** [comment]
Which tracking mode are you using? libobjecttracker or motioncapture? In principle what you describe should work in both modes.

From the error, it looks like the server is getting a segfault. The best way to investigate is to attach a debugger to find the line where the error occurs.
